doc/starterkit/k4MarlinWrapperCLIC/CEDViaWrapper.md

```diff
@@ -102,13 +102,17 @@ ApplicationMgr( TopAlg = algList,
               )
 ```
 
-With all these pieces put together (as in `examples/event_display.py`) it is no possible to run the event display. In order to run the event display we first have to start the `glced` server program to which the wrapped `CEDViewer` processor will then connect. Starting the server and running the wrapped processor can be done via
+With all these pieces put together (as in `examples/event_display.py`) it is now possible to run the event display. In order to run the event display we first have to start the `glced` server program to which the wrapped `CEDViewer` processor will then connect. Starting the server and running the wrapped processor can be done via
 ```bash
 glced &
 
-k4run $K4MARLINWRAPPER/examples/event_display.py --EventDataSvc.input=gamma_10GeV_edm4hep.root
+k4run $K4MARLINWRAPPER/examples/event_display.py --EventDataSvc.input=gamma_10GeV_edm4hep.root || true
```

Comment:
I would prefer this to be opt-in: we give the command without `|| true` and then an explanation on why you may want to use it if running in a test or something like that, but that's kind of expert knowledge that is unlikely to be useful for most people

Reply:
The problem is that these bits of the documentation are run in the `doctest` workflow and without any handling of the non-zero exit code from `k4run` they will fail.